.github/test-a-feature.md

```diff
@@ -103,4 +103,49 @@ yarn test
 yarn test:watch
 ```
 
-[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
\ No newline at end of file
+[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
+
+## Screener Tests
+
+For some components, it is necessary to write screenshot tests in order to check they render properly. For each component added to the docsite, a screenshot test is automatically created. This checks if that the component is rendered in a consistent way, as it checks the visual differences between the previous and the current rendering. We use [screener-io](https://screener.io/) to achieve this.
+
+### Tests with Steps API
+
+This default test only checks the rendering for the component in its initial state. In order to test the rendering of more complex components, such as a `Dropdown`, screener provides an [api](https://www.npmjs.com/package/screener-runner) to execute actions on the DOM, in a way similar to end-to-end tests. These tests are automatically sent to Screener in our test runs, as long as the tests and their files respect the conventions:
+- the test file should be placed at the same location as the tested component in the docs.
```

Comment:
In my opinion it feels kind a awkward that we are adding tests (steps file) in the docs rather then in the test folder, where all other tests resides. I understand that the initial components are there, but in my opinion maybe it will be worth it to just initialize the component in the same place where the test will reside. Any strong reason for not doing this?

Reply:
Btw, the comment has nothing to do with the docs, just expressing my thoughts on the procedure

Reply:
We are testing an existing example, it's very useful that we have visual tests in the same place 👀

Reply:
I would not agree with this one, especially that it is in the docs. If we decide that we don't want to show some example, someone can accidentally remove it, not knowing it was used in testing (yeah, we have the same steps.ts file there as an indicator, and we can remove it just from the ExampleSection) but still, coupling the docs site with tests seems like a red light to me 👀 (watching you too) :D

Reply:
> If we decide that we don't want to show some example, someone can accidentally remove it, not knowing it was used in testing

How is that different from current behavior? We already have visual tests and removing an example causes screener tests to fail. It's been like this from the beginning.

> coupling the docs site with tests seems like a red light to me
It may be but once again, aren't we doing this already? :) It's just that now we're adding extra steps to already existing tests. At least by having the files here the developer will be aware that THERE ARE some visual tests running.

Also, there is not as if there is something that is going to fail, we'll just have screen regression (nothing new)

@mnajdova what is the alternative in the current state of the project?
The long term solution is E2E tests but I don't know where that is on the list in terms of priority

Reply:
Yep, we are doing it already, but that doesn't mean it's the right thing to do. At least these files, are actually used in the docs, as examples for the users. I guess the initial thought was, let's reuse these files, so till some point I would understand that (we may change this in the future). But now, we are actually adding test steps in the same directory (that are not in any ways used in the docs). The alternative is, we can define this files in the tests directory, and reuse the files from the docs. Then if we decide that for the purpose of the tests we need to change the initial screen, we can create a separate file, that will not necessarily need to be part of the docs. Anyways, I started the discussion with asking whether there is strong reason for not splitting the tests in a separate place (and apparently from the comments, there isn't). Anyways, we can iterate on this later, and not block the docs which is the main focus for this PR :)